src/main/java/org/broadinstitute/hellbender/tools/AnalyzeSaturationMutagenesis.java

```diff
@@ -683,6 +689,7 @@ private int updateCoverage( final List<Interval> refCoverageList ) {
         public void bumpNWildType() { nWildType += 1; }
         public long getNWildType() { return nWildType; }
         public void bumpInconsistentPairs() { nInconsistentPairs += 1; }
+        public void bumpInconsistentPairs( final int nReads ) { nInconsistentPairs += nReads; }
```

Comment:
I'm a little concerned about downstream interpretation: would the report indicate (via header or whichever means) that the analysis was performed with inconsistent pairs reported separately?
Because otherwise, this is inconsistent, for the function above only bumps the pair count by 1, whereas this overload&mdash;judged by its usage on line 1635&mdash;increases the count by 2, which is a little counter-intuitive for a definition of `pair`.

Reply:
I think you have a point.  The numbers are right:  the sum of molecules in the various categories adds up to the total number of reads.  But someone will surely be confused about how inconsistent pairs are counted in this mode (i.e., as 2 inconsistent reads).  I'll try to figure out what to do.

Reply:
BTW, thanks for picking up this review.  I don't know if this code will be needed.  It's being debated.